dali/test/python/test_operator_audio_resample.py

```diff
@@ -0,0 +1,68 @@
+# Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
Please add this a test to test_dali_cpu_only.py and test_dali_variable_batch_size.py.